src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -178,3 +178,8 @@ def test_vi_int32_output_function(session):
     assert isinstance(backlog, int)
     assert backlog == 0
 
+
+def test_initialize_nondefault():
+    with nidmm.Session('FakeDevice', False, False, 'Simulate = 1,RangeCheck=0,QueryInstrStatus=1,Cache=0,DriverSetup=Model:4080; BoardType:PXIe') as session:
+        assert session.simulate is True
+

```

Comment:
This seems like it could be a unit test right? The defaults are set by __init__ in generated code. Additionally, It seems that False, False is the default for the python code. (excluding resource name and option string)

    def __init__(self, resource_name, id_query=0, reset_device=False, options_string=""):
        self.library = library_singleton.LibrarySingleton.get()
        self.vi = 0  # This must be set before calling _init_with_options.
        self.vi = self._init_with_options(resource_name, id_query, reset_device, options_string)

Reply:
I think it is possible.
Unit tests would mean that this would go into the niFake folder right?
I noticed that session.simulate etc were not available under niFake